AmericanExpress/build.gradle

```diff
@@ -15,6 +15,15 @@ android {
         testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
     }
 
+    signingConfigs {
```

Comment:
Not sure if there is a way to do this, but is there a way we can define this like how we have `compileSdkVersion` defined in the top level `build.gradle` that then every other `build.gradle` references? Just thinking about forgetting 1 of these files if we ever need to make an update and would be nice to make the change in 1 place that everything else references.

Reply:
That's a good call. I believe we're only defining simple properties to share across our gradle files. So defining the signing configs block would be different. I'm no groovy expert, but let me see if I can find the syntax for it.

Reply:
Adding the `android { ... }` blcok to the gradle-publish.gradle file ended up working!